ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -1485,6 +1485,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-ultrassd-disks-p3-f28
+  cron: 3 12 30 * *
+  steps:
+    cluster_profile: azure-qe
```

Comment:
Missing the following configurations for arm:
```
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```

Reply:
added to 4.12 and 4.14 as well , thanks!